triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/LoadTesting.scala

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.engine.trigger.test
+
+import akka.stream.scaladsl.Flow
+import com.daml.ledger.api.testing.utils.SuiteResourceManagementAroundAll
+import com.daml.ledger.api.v1.commands.CreateCommand
+import com.daml.ledger.api.v1.event.Event.Event.Created
+import com.daml.ledger.api.v1.event.{Event => ApiEvent}
+import com.daml.ledger.api.v1.transaction.{Transaction => ApiTransaction}
+import com.daml.ledger.api.v1.{value => LedgerApi}
+import com.daml.ledger.runner.common.Config
+import com.daml.ledger.sandbox.SandboxOnXForTest.{ApiServerConfig, singleParticipant}
+import com.daml.lf.data.Ref.QualifiedName
+import com.daml.lf.engine.trigger.Runner.TriggerContext
+import com.daml.lf.engine.trigger.{
+  ACSOverflowException,
+  InFlightCommandOverflowException,
+  TriggerMsg,
+  TriggerRunnerConfig,
+}
+import com.daml.platform.services.time.TimeProviderType
+import com.daml.util.Ctx
+import org.scalatest.{Inside, TryValues}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AsyncWordSpec
+
+import scala.concurrent.Future
+
+abstract class LoadTesting
```

Comment:
Using an abstract class with concrete implementing classes in this file allows for tests to have independent trigger configs